run_validation/main_task/sample_run.json

```diff
@@ -0,0 +1,5336 @@
+{
```

Comment:
Put this in a sample data directory overall rather than in a the per-task folder.